build/phase0/js/amd_loader.js

```diff
@@ -2,7 +2,9 @@ var define, requirejs;
 
 // NOTE(joe): this is defined for use in dependency bundles.  Even though it
 // isn't used in this file, it's an important definition
-var nodeRequire = require;
+if(typeof require === "function") {
+  var nodeRequire = require;
+}
```

Comment:
I can imagine a few reasons for this – what made it be necessary?

Reply:
Hmmm, I'm unsure. That came from running new-bootstrap, so it's not from any code that I wrote.

Reply:
Oooh I misread. Got it. Going to have to be careful with reviewing this new phase0 "standard library" stuff.